src/crate/client/connection.py

```diff
@@ -28,7 +28,9 @@
 
 class Connection(object):
     def __init__(self, servers=None, timeout=None, client=None,
-                 verify_ssl_cert=False, ca_cert=None, error_trace=False):
+                 verify_ssl_cert=False, ca_cert=None,
+                 cert_file=None, key_file=None,
```

Comment:
could you add the args as last params in order to not break backward compatibility for people who might have used positional arguments?